pkg/apis/hubcluster/v1alpha1/types.go

```diff
@@ -0,0 +1,299 @@
+package v1alpha1
```

Comment:
kubebuilder puts the API types in the `api` directory at the root of the repo. The location of this file is the location I'm normally used to, but it does differ from where kubebuilder puts things by default.